lib/support/chrome-perflog-parser.js

```diff
@@ -312,9 +312,10 @@ function populateEntryFromResponse(entry, response, timestamp) {
     } else {
       entry.request.headersSize = calculateRequestHeaderSize(entry.request);
     }
-
-    const cookieHeader = response.requestHeaders.Cookie || '';
-    entry.request.cookies = parseCookies(cookieHeader.split(';'));
+    if(response.hasOwnProperty('requestHeaders') && response.requestHeaders.hasOwnProperty('Cookie')) {
```

Comment:
Only the check for requestHeaders is really needed here. If Cookie is missing, cookieHeader will be set to empty string on the next line.